### PR TITLE
Forward slash path input fix

### LINKED ISSUE #9175: path accepts only \ and not /
### Description

When I use "/" instead of "\\" in the path input, the folder name shows differently on the tab header and the folder icon doesn't appear. But I can access the folder, which seams kind of odd.
Example:
> Path => C:\Users\User\Desktop\Path/To/Folder
> Tab title => Path/To/Folder (without icon)

I also tried to use drag & drop with winrar (from Files to Winrar) and it didn't work until I changed all '/' in the path to '\\'
I haven't tried to go to a path with '/' in the middle of it, just at the end of the path like shown in the example

### Steps To Reproduce

1. Open Files
2. Enter a path manually with both '\\' and '/'

### Expected behavior

- Tab title shows the folder name only with its icon
- Drag & drop works properly

### Files Version

Version: 2.1.24.0 OS Version: 10.0.19044.1706

### Windows Version

Windows 10 Pro 21H2 19044.1706 + Windows Feature Experience Pack 120.2212.3030.0

### Relevant Assets/Logs

[debug.log](https://github.com/files-community/Files/files/8694517/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/8694518/debug_fulltrust.log)

### PR DESCRIPTION
**Pull request by puppetsw**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #9175

**Details of Changes**
Add details of changes here.
- Any forward slash input in path box is replaced with escaped backslashes.

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

File: src/Files.Uwp/ViewModels/ToolbarViewModel.cs [hunk 1]
```diff
@@ -915,6 +915,11 @@ public async Task SetPathBoxDropDownFlyoutAsync(MenuFlyout flyout, PathBoxItem p
 
         public async Task CheckPathInput(string currentInput, string currentSelectedPath, IShellPage shellPage)
         {
+            if (currentInput.Contains("/"))
+            {
+                currentInput = currentInput.Replace("/", "\\", StringComparison.Ordinal);
```
Comment (gave92) [inline src/Files.Uwp/ViewModels/ToolbarViewModel.cs:920]: Could you check if this creates issues when browsing an FTP server?

Comment (puppetsw) [reply src/Files.Uwp/ViewModels/ToolbarViewModel.cs:920]: That’s a good thought. I didn’t consider that. I’ll get back to you.

Comment (puppetsw) [reply src/Files.Uwp/ViewModels/ToolbarViewModel.cs:920]: Yeah it creates issues for ftp paths. Maybe I'll close this one for now and have another look.

Comment (gave92) [reply src/Files.Uwp/ViewModels/ToolbarViewModel.cs:920]: Thanks, can't test right know but commit 944596c looks good.

**Review by gave92 — APPROVED** [review]

**Pull request merged**